.babelrc

```diff
@@ -16,7 +16,14 @@
 	],
 	"plugins": [
 		"@babel/plugin-syntax-dynamic-import",
-		"@babel/plugin-syntax-import-meta"
+		"@babel/plugin-syntax-import-meta",
+		"@babel/plugin-transform-modules-commonjs",
+		[
+			"@babel/plugin-transform-runtime",
+			{
+				 "regenerator": true
```

Comment:
This is in response to an error I was getting from the AsYouTypeFormatter in libphonenumber-js (in KvPhoneInput.vue). 

I'm really not sure what the implications of this are in prod. Are there performance concerns?

![image](https://user-images.githubusercontent.com/187937/102131357-f0772c80-3e06-11eb-98e7-42dfaa60015a.png)

Reply:
This should be fine. I think it will actually reduce the bundle size a little bit, if I'm correctly understanding what it does.

Reply:
Ive never run into this. Looks ok 👍